functions/Test-DbaBackupInformation.ps1

```diff
@@ -148,6 +148,34 @@ Function Test-DbaBackupInformation {
 						}
 					}
 				}
+				#Easier to do FileStream checks out of the loop:
+				if ('s' -in ($DbHistory | Select-Object -ExpandProperty filelist | Select-Object Type -Unique).Type) {
```

Comment:
This should be `FileType` instead of `Type`

Reply:
Missing this two. Change to `FileType` (instead of 'Type`)
Otherwise is good to go!